share/completions/passwd.fish

```diff
@@ -11,18 +11,20 @@ if passwd --help >/dev/null 2>&1
     complete -c passwd -n '__fish_contains_opt -s S status' -s a -l all -f -d "Display password state for all users"
     complete -c passwd -s d -l delete -f -d "Delete user password"
     complete -c passwd -s e -l expire -f -d "Immediately obsolete user password"
+    complete -c passwd -s f -l force -f -d "Force the specified operation"
```

Comment:
which version of passwd are you running?
I have shadow-utils 4.8.1 here, where
- there's no -f/--force or --stdin
- --minimum/--maximum/--warning are still called --mindays/--maxdays/--warndays

Reply:
Interesting. On my Fedora system, `passwd` is not part of `shadow-utils` but

```
> dnf provides (which passwd)
Last metadata expiration check: 0:02:37 ago on So 07 Nov 2021 10:31:42 CET.
passwd-0.80-11.fc35.x86_64 : An utility for setting or changing passwords using PAM
Repo        : @System
Matched from:
Filename    : /usr/bin/passwd

passwd-0.80-11.fc35.x86_64 : An utility for setting or changing passwords using PAM
Repo        : fedora
Matched from:
Filename    : /usr/bin/passwd


> dnf info passwd
Last metadata expiration check: 0:02:43 ago on So 07 Nov 2021 10:31:42 CET.
Installed Packages
Name         : passwd
Version      : 0.80
Release      : 11.fc35
Architecture : x86_64
Size         : 431 k
Source       : passwd-0.80-11.fc35.src.rpm
Repository   : @System
From repo    : fedora
Summary      : An utility for setting or changing passwords using PAM
URL          : https://pagure.io/passwd
License      : BSD or GPL+
Description  : This package contains a system utility (passwd) which sets
             : or changes passwords, using PAM (Pluggable Authentication
             : Modules) library.
```

Maybe Fedora/RedHat's own implementation. Maybe we should close this PR then if this not the usually found `passwd`.